tests/stress/scopez_server/scopez_stress_test.py

```diff
@@ -0,0 +1,279 @@
+        if '200' in a_results:
+            a_results['200'] += 1
+        else:
+            a_results['200'] = 1
+    if l_code >= 300 and l_code < 400:
+        if '300' in a_results:
+            a_results['300'] += 1
+        else:
+            a_results['300'] = 1
+    if l_code >= 400 and l_code < 500:
+        if '400' in a_results:
+            a_results['400'] += 1
+        else:
+            a_results['400'] = 1
+    if l_code >= 500 and l_code < 600:
+        if '500' in a_results:
+            a_results['500'] += 1
+        else:
+            a_results['500'] = 1
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def post_config(a_host, a_template, a_type, a_idx):
+    if isinstance(a_template, list):
+        for l_instance in a_template:
+            if 'last_modified_date' in l_instance:
+                l_instance['last_modified_date'] = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+            if 'name' in a_template:
+                l_instance['name'] = str(a_idx);
+    else:
+        if "last_modified_date" in a_template:
+            a_template['last_modified_date'] = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        if 'name' in a_template:
+            a_template['name'] = str(a_idx);
+    l_headers = {}
+    l_headers['Content-type'] = 'application/json'
+    l_url = '%s/update_%s'%(a_host, a_type)
+    l_body = json.dumps(a_template)
+    # ------------------------------------------------------
+    # POST
+    # ------------------------------------------------------
+    try:
+        l_rq = Request(l_url, l_body.encode(), l_headers)
+        l_r = urlopen(l_rq, timeout=20.0)
+    except Exception as l_e:
+        print('error: performing POST to %s. Exception: %s' % (l_url, l_e))
+        sys.exit(1)
+    l_body = l_r.read()
+    if l_r.getcode() != 200:
+        print('error: performing POST to %s -status: %d. Response: %s' % (l_url, l_r.getcode(), l_body))
+        sys.exit(1)
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def scopez_server_stress(a_verbose,
+                         a_port,
+                         a_template,
+                         a_type,
+                         a_vector_file,
+                         a_num_ids):
+    global g_run
+    l_host = 'http://127.0.0.1:%d'%(a_port)
+    # ------------------------------------------------------
+    # read template
+    # ------------------------------------------------------
+    l_template = []
+    try:
+        with open(a_template) as l_f:
+            l_template = json.load(l_f)
+    except Exception as l_e:
+        print('error opening template file: %s.  Reason: %s error: %s, doc: %s' % (
+            a_vector_file, type(l_e), l_e, l_e.__doc__))
```

Comment:
arg for `template file: %s` should be `a_template`

Reply:
fixed